frontend/packages/dev-console/console-extensions.json

```diff
@@ -653,6 +653,21 @@
       }
     }
   },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "topology",
+      "perspective": "admin",
+      "section": "workloads",
+      "name": "%devconsole~Topology%",
+      "href": "/topology",
+      "namespaced": true,
+      "dataAttributes": {
+        "data-quickstart-id": "qs-nav-topology",
```

Comment:
This makes me realize that we should look at the quick starts to see what changes are needed for the combined perspective. We should track that as a story under the convergence epic.

Reply:
Sure I will create a story for a quick start under epic [ODC-7716](https://issues.redhat.com/browse/ODC-7716)